### ISSUE TITLE
Better errors when #lang racket is missing

### ISSUE THREAD

**Issue by martinklepsch** [greghendershott/racket-mode#160]
When there's no `#lang racket` at the beginning of the file `racket-run` will fail with something like this:

```
; module->language-info: unknown module in the current namespace
;   name: #<resolved-module-path:(submod "/Users/martin/code/glitter/util.rkt" util)>
```

Took me some time to figure out. As soon as I add `#lang racket` things worked smoothly!

**Comment by greghendershott** [comment]
Thanks for reporting this in IRC -- and thanks even more for taking the time to log an issue here!

This is interesting because

```
#lang racket
```

is actually a shorthand for

```
(module filename racket ___)
```

The latter is valid. In fact once upon a time, there was no `#lang` line, and this was the only way. You just don't see it much anymore in new Racket code.

I recently added support for Racket submodules. `racket-run` will enter the submodule around point, if any, otherwise run the file module. However I forgot to handle the case where the file has no `#lang` and instead consists of a single `module` form. That's not a _sub_ module. It's _the_ module for the file.

TL;DR Although it's probably not what you intended, your example is actually a valid Racket file. However I misidentified it as a submodule, and the confusing error message is the result.

I have a commit to sort this out correctly (I hope).

**Comment by martinklepsch** [comment]
Thanks a lot for the educating response :)
